docs/src/tutorials/bvp_example.md

```diff
@@ -64,7 +64,12 @@ We have used the mono-implicit Runge–Kutta (MIRK) method to solve the BVP, but
 ```julia
 using OrdinaryDiffEq
 u₀_2 = [-1.6, -1.7] # the initial guess
-sol3 = solve(bvp1, Shooting(Vern7()))
+function bc3(residual, u)
```

Comment:
You should explain here that for `Shooting`, `u` is the "current solution". In fact, you may want to also change the name to `sol` to make that more explicit.